snapshotter/create_snapshot.sh

```diff
@@ -27,15 +31,15 @@ fi
 
 
 # Initial sanity checks passed. Continuing on with a snapshot.
-docker exec miner miner snapshot take /var/data/saved-snaps/latest
-BLOCK_HEIGHT=$(docker exec miner miner snapshot info /var/data/saved-snaps/latest | head -1 | awk {'print $2'})
-BLOCK_HASH_PART1=$(docker exec miner miner snapshot info /var/data/saved-snaps/latest | tail -3 | head -1 | awk {'print $2'})
-BLOCK_HASH_PART2=$(docker exec miner miner snapshot info /var/data/saved-snaps/latest | tail -2 | head -1 | awk {'print $1'})
+docker exec miner miner snapshot take /var/data/saved-snaps/latest || sleep 180
+BLOCK_HEIGHT=$(docker exec miner miner snapshot info /var/data/saved-snaps/latest.gz | head -1 | awk {'print $2'})
+BLOCK_HASH_PART1=$(docker exec miner miner snapshot info /var/data/saved-snaps/latest.gz | tail -3 | head -1 | awk {'print $2'})
+BLOCK_HASH_PART2=$(docker exec miner miner snapshot info /var/data/saved-snaps/latest.gz | tail -2 | head -1 | awk {'print $1'})
 BYTE_ARRAY="${BLOCK_HASH_PART1}${BLOCK_HASH_PART2}"
 BASE64URL_FORMAT=$(python3 /home/snapshot/hm-block-tracker/snapshotter/base64url_encoder.py "$BYTE_ARRAY")
 TMPDIR=$(mktemp -d)
 
-mv /var/miner_data/saved-snaps/latest "$TMPDIR/snap-$BLOCK_HEIGHT"
+mv /var/miner_data/saved-snaps/latest.gz "$TMPDIR/snap-$BLOCK_HEIGHT"
```

Comment:
@marvinmarnold Do you remember what was the end result of the question "whether our url should have .gz in the file name or not". I remember working with the gz confusion in the past, but the problem is still here.

It is possible that last time they were writing the given filename as gz and this time they are attaching the .gz extension as well.

Reply:
At this stage snapshot is working but I am not sure whether the filename in the url should also have .gz or not.

Reply:
We need to save the snapshot as .gz as the miner by default is looking for a .gz file when downloading the snapshot. But it doesn't actually need to be compressed

Reply:
Yeah, I looked at master, it already has changs we did last month and .gz is included. Somehow VM didn't have those.